--- Cargo.lock.before
+++ Cargo.lock.after
@@ -1371,7 +1371,6 @@
 [[package]]
 name = "nostd-print"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "syscalls",
 ]
@@ -1709,7 +1708,6 @@
 [[package]]
 name = "reverie-core"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "addr2line",
  "anyhow",
@@ -1735,7 +1733,6 @@
 [[package]]
 name = "reverie-dbt"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "bincode 2.0.1",
  "libc",
@@ -1753,7 +1750,6 @@
 [[package]]
 name = "reverie-kvm"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "futures 0.3.33",
  "goblin",
@@ -1768,7 +1764,6 @@
 [[package]]
 name = "reverie-liteinst"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "bincode 2.0.1",
  "libc",
@@ -1786,7 +1781,6 @@
 [[package]]
 name = "reverie-memory"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "syscalls",
 ]
@@ -1794,7 +1788,6 @@
 [[package]]
 name = "reverie-preload"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "bincode 2.0.1",
  "libc",
@@ -1805,7 +1798,6 @@
 [[package]]
 name = "reverie-process"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "bincode 2.0.1",
  "bitflags 2.13.1",
@@ -1822,7 +1814,6 @@
 [[package]]
 name = "reverie-ptrace"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "anyhow",
  "async-trait",
@@ -1853,7 +1844,6 @@
 [[package]]
 name = "reverie-rpc"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "async-trait",
  "bincode 1.3.3",
@@ -1864,7 +1854,6 @@
 [[package]]
 name = "reverie-rpc-macros"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "darling",
  "heck 0.3.3",
@@ -1876,7 +1865,6 @@
 [[package]]
 name = "reverie-rpc-transport"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "async-trait",
  "bincode 2.0.1",
@@ -1890,7 +1878,6 @@
 [[package]]
 name = "reverie-sabre"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "atomic 0.5.3",
  "cc",
@@ -1915,7 +1902,6 @@
 [[package]]
 name = "reverie-sabre-macros"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "darling",
  "heck 0.3.3",
@@ -1927,7 +1913,6 @@
 [[package]]
 name = "reverie-sabre-stats"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "libc",
  "reverie-core",
@@ -1936,7 +1921,6 @@
 [[package]]
 name = "reverie-syscalls"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "bitflags 2.13.1",
  "derive_more",
@@ -2000,7 +1984,6 @@
 [[package]]
 name = "safeptrace"
 version = "0.2.0"
-source = "git+https://github.com/rrnewton/reverie.git?rev=114b309413612fafc2657c74e83811c71aac7b19#114b309413612fafc2657c74e83811c71aac7b19"
 dependencies = [
  "bitflags 2.13.1",
  "futures 0.3.33",
